src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -273,6 +273,14 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
                     _lazyParameters = CustomModifierUtils.CopyParameterCustomModifiers(overriddenOrImplementedProperty.Parameters, _lazyParameters, alsoCopyParamsModifier: isOverride);
                 }
             }
+            else if (_refKind == RefKind.RefReadOnly && (this.IsVirtual || this.IsAbstract))
```

Comment:
I think we need some tests for interface implementations

---
In reply to: [117863982](https://github.com/dotnet/roslyn/pull/19658#discussion_r117863982) [](ancestors = 117863982)